### PR TITLE
Adds the ability to override the default respawn timers via a config value

### PR DESCRIPTION
**Pull request by ZephyrTFA**
Adds config values to override the respawn timers for both marines and xenomorphs which are read on the Ticker's Initialize.

MARINE_RESPAWN and XENO_RESPAWN are now config values that you can set to override the default respawn timers.
Hard-coded values are bad and this lets the values be changed globally without needing to be manually changed ingame.

## Changelog
:cl:
server: Two new config values for overriding the respawn timers of both marines and xenomorphs.
/:cl:

### CODE REVIEW

**Comment by BraveMole** [comment]
If the plan is to make it so they can be tweaked depending on the gamemode, then no, don't do that

**Comment by ZephyrTFA** [comment]
> 
> 
> If the plan is to make it so they can be tweaked depending on the gamemode, then no, don't do that

The idea is that you can change them via a config value. End of story.

**Comment by BraveMole** [comment]
oh, i thought you were xenophile, my bad

File: code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -41,6 +41,9 @@ SUBSYSTEM_DEF(ticker)
 
 /datum/controller/subsystem/ticker/Initialize(timeofday)
 	load_mode()
+	
+	GLOB.respawntime = CONFIG_GET(number/marine_respawn)
+	GLOB.xenorespawntime = CONFIG_GET(number/xeno_respawn)
```
Comment (SplinterGP) [inline code/controllers/subsystem/ticker.dm:46]: i think its better to replace all GLOB.respawntime and GLOB.xenorespawntime for the CONFIG_GET

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/controllers/subsystem/ticker.dm (hunk 1 above)
Comment (TiviPlus) [reply code/controllers/subsystem/ticker.dm:46]: No this is fine

**Review by TiviPlus — APPROVED** [review]

File: code/controllers/subsystem/ticker.dm (hunk 1 above)
Comment (SplinterGP) [reply code/controllers/subsystem/ticker.dm:46]: Oh ye thats right

**Review by SplinterGP — APPROVED** [review]

**Pull request merged**